models/tmp/stg_recharge__address_shipping_line_tmp.sql

```diff
@@ -1,5 +1,3 @@
 
-select
-    *
-from
-    {{ var('address_shipping_line') }}
\ No newline at end of file
+select *
+from {{ var('address_shipping_line') }}
```

Comment:
A small style update request. Can we have the tmp models follow this style structure. Just to be consistent with our other packages.

Reply:
Done.